src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Validator.cs

```diff
@@ -1,6 +1,8 @@
 ﻿using FastEndpoints;
 using FluentValidation;
 
+// ReSharper disable ArrangeRedundantParentheses
```

Comment:
What is this for?

Reply:
On save it takes out the parenthesis, and then it errors saying I need them. And then when I put them in, it takes them out on save, and on and on.